test/test_bank_acc.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+from challenges.bank_account import BankAccount
+
+
+class TestBank(unittest.TestCase):
+    def test_deposit_func(self):
+        user = BankAccount("Amit")
+        user.deposit(100)
+        self.assertEqual(user.bal, 100)
+
+        user = BankAccount("Amit")
+        user.deposit(-100)
+        self.assertEqual(user.bal, 0)
+
+        user = BankAccount("Amit", 200)
+        user.deposit(-100)
+        self.assertEqual(user.bal, 200)
+
+        user = BankAccount("Amit")
+        user.deposit(0)
+        self.assertEqual(user.bal, 0)
+
+        user = BankAccount("Amit", 100)
+        user.deposit(0)
+        self.assertEqual(user.bal, 100)
+
+    def test_withdraw_func(self):
```

Comment:
The general convention for this is to name test function this way - `test_withdraw` which is test_ followed by function name